section-engineering-education/engineering-education
# Page views per article

Display number of page views for each article on the authors bio page.

**How to Test:**
Navigate to beta.section.io/engineering-education/ (or pull down locally), and visit a few author bio pages. You should see the new page views icon with the number of page views at the bottom of each article listed in the authors bio page (screenshot below).

<img width="383" alt="image" src="https://user-images.githubusercontent.com/15935329/161638583-c8c9af28-51e1-4deb-bc51-e5fa79936233.png">

If interested the API repo is [here](https://github.com/section-io/ga-page-views-app).
